<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(47, 57, 69);
} </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(47, 57, 69);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid grey;
background-color:rgb(255, 255, 255);

  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>№ Города:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>621</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView{
border: none;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optim">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>440</y>
      <width>306</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid grey;
background-color:rgb(255, 255, 255);

  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gorod">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid grey;
background-color:rgb(255, 255, 255);

  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>440</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid grey;
background-color:rgb(255, 255, 255);

  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Путь:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_put">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>440</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 1px solid grey;
background-color:rgb(255, 255, 255);

  qproperty-alignment: 'AlignVCenter | AlignCenter';
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
color: white;
background-color: rgb(54, 64, 79);
border: 4px solid rgb(95, 113, 139);
border-radius: 24px;
}

QPushButton::pressed
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(54, 64, 79, 255), stop:1 rgba(170, 170, 170, 255));
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>label</zorder>
   <zorder>label_optim</zorder>
   <zorder>label_gorod</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_2</zorder>
   <zorder>label_put</zorder>
   <zorder>pushButton_7</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
